cg/utils/files.py

```diff
@@ -1 +1,22 @@
 """Some helper functions for working with files"""
+from pathlib import Path
+from typing import Optional
+import os
+
+
+def get_file_in_directory(directory: Path, file_name: str) -> Path:
+    """Get a file in a directory and subdirectories.
+    Raises:
+        FileNotFoundError: If the file is not found in the directory.
+        FileNotFoundError: If the directory does not exist.
+    """
+    if not directory.is_dir() or not directory.exists():
+        raise FileNotFoundError(f"Directory {directory} does not exist")
+    path_to_file: Optional[Path] = None
+    for directory_path, _, files in os.walk(directory):
+        for file in files:
+            if file_name in file:
```

Comment:
This might be fine, just wondering whether we want an exact match or not? If so it should be `file_name == file`